app/jobs/retrieve_award_job.rb

```diff
@@ -7,10 +7,15 @@ class RetrieveAwardJob < ApplicationJob
 
   class TraineeAttributeError < StandardError; end
 
-  def perform(trainee, timeout_after)
+  def perform(trainee, timeout_after = nil)
     @timeout_after = timeout_after
     @trainee = trainee
 
+    if @timeout_after.nil?
+      self.class.perform_with_default_delay(trainee)
```

Comment:
I don't think we can use the `default_delay` method here. Well, we could be we might end up with a job that is say 3 days into polling being set to poll for another 4 days. I think we need - 

`self.class.perform(trainee, trainee.submitted_for_trn_at + Settings.jobs.Settings.jobs.max_poll_duration_days.days)`

Maybe it's not a big deal for the small number of jobs that are queued with the old method signature.